### PR TITLE
Throw out

### PR DESCRIPTION
**Pull request by rebelz-il**
added parameter to not throw error after capturing.
added sentry debug support

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -235,16 +249,24 @@ function wrapCallback(pluginConfig, cb) {
...
+		if (err) {
+			if (pluginConfig.captureErrors) {
+				const Raven = pluginConfig.ravenClient;
+				const captureOptions = {};
+				debug("capturing exception with options", captureOptions);
+				ravenInstalled && Raven.captureException(err, captureOptions, () => {
+					cb(throwOut, data);
+				});
+			}
+			else {
```
Comment (elad-haz) [inline src/index.js:263]: Why split the if to two if?

File: src/index.js [hunk 2]
```diff
@@ -432,39 +455,45 @@ class RavenLambdaWrapper {
...
-								const Raven = pluginConfig.ravenClient;
-								return new Promise((resolve, reject) => Raven.captureException(err, {}, () => {
-									reject(err);
-								}));
-							}
-							else {
-								return Promise.reject(err);
-							}
-						});
+							.then((...data) => {
```
Comment (elad-haz) [inline src/index.js:461]: Need explanation on the throwOut

Comment (rebelz-il) [reply src/index.js:461]: here? or in general in the file?

File: src/index.js (hunk 1 above)
Comment (rebelz-il) [reply src/index.js:263]: two different debug messages

File: src/index.js (hunk 2 above)
Comment (elad-haz) [reply src/index.js:461]: In general

**Review by elad-haz — APPROVED** [review]

**Pull request closed without merging**